README.rst

```diff
@@ -85,6 +85,11 @@ Details of the config.yaml file are in `edx-platform/conf/locale/config.yaml
 
 Changes
 =======
+v0.3.11
```

Comment:
I don't want to get too pedantic, but since the point of this change is to change the external behavior of the program, the version number should be 0.4

Reply:
Changed.